internal/backend/dispatcher/special_triggers.go

```diff
@@ -35,6 +35,15 @@ func processHTTPTrigger(trigger sdktypes.Trigger, event sdktypes.Event) (bool, m
 
 	triggerPath := pathValue.GetString().Value()
 
+	if triggerPath == "" {
+		// Empty path means we don't need to match anything.
+		return true, nil, nil
+	}
+
+	if triggerPath[0] != '/' {
```

Comment:
it is a bit low level but this only works since '/' is represented as 1 byte in utf-8, otherwise the comparison would fail

probably better to use something like
```go
if !strings.HasPrefix(triggerPath, "/") {
...
}
```

Reply:
fixed